setup.py

```diff
@@ -58,7 +58,8 @@ def main():
             'Pygame>=1.9.4',
             'pygame-menu>=2.0.4',
             'psutil>=5.5.1',
-            'pluggy>=0.13.1'
+            'pluggy>=0.13.1',
+            'gpiozero>=1.5.1'
```

Comment:
You need to put a filter to avoid the installation on non compatible devices, and maybe remove Rpi.GPIO from the setup if it is not used.

Reply:
Agree, remove the `Rpi.GPIO`, the dependency is included in `gpiozero`

Reply:
Still need to avoid installation of gpiozero on non compatible devices

Reply:
gpiozero can be installed on any devices, this is the deal ;)

Reply:
Ok I see but I thought the whole point of switching to gpiozero was to not use Rpi.GPIO, but we still use it in booth.py...

I need more time to look deeper into it.

Reply:
gpiozero is a wrapper around several GPIO libs, Api.GPIO is one of them.
Thus, yes it's still used, but it is hidden.